src/datapoints-graph/alarm-event-selector/alarm-event-selector.model.ts

```diff
@@ -30,13 +30,15 @@ export type AlarmDetails = AlarmOrEventBase & {
   filters: {
     type: string;
   };
+  [key: string]: any;
```

Comment:
Because of the __hidden property which is needed to show or hide the events/alarms on the chart.

Reply:
why not adding '__hidden' key to type declaration (as optional prop)? scope of new properties is finite and known, so it's more beneficial to have it  (e.g. it would detect typos in accessing this property- right now, if I try to write `alarmOrEvent._hidden`, TS won't complain)

Reply:
added